encryption/src/main/java/com/diyasylum/encryption/EncryptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.diyasylum.encryption;
+
+import com.diyasylum.encryption.models.EncryptedMessage;
+import com.diyasylum.encryption.models.EncryptionPackage;
+import com.diyasylum.encryption.models.UnencryptedMessage;
+
+public class EncryptionHandler {
+  // Is there a better way to handle missing values here / propagate errors?
```

Comment:
I guess I was thinking more about propagating knowledge of the problem through the API to the FE. But we don't have a real use for that yet anyway.

Reply:
Status codes?

Reply:
In some cases I'd have more detailed knowledge which I could hypothetically pass along. Like 400 + no content + the salt you gave me wasn't the right length. But that's going overboard and isn't really relevant. For now I'll just throw some errors and call it a day.

Reply:
Well, this could throw a custom exception with a message. The actual endpoint code would catch the exception and return the appropriate status code with the message as the body. Or just wait till the need arises

Reply:
Turns out Spring has this nice annotation @ExceptionHandler which you can use to send meaningful error messages when exceptions are thrown. Using that now.